Test/Integration/_files/order.php

```diff
@@ -162,6 +162,7 @@
     $options[] = $option;
 }
 
+$options = []; // TODO figure out the crash with the options, are they necessary for this test? No assertions on them
```

Comment:
I am not sure the intention of the previous options in the scope of the existing test.

They do not seem to be asserted against at all.